homeassistant/components/luxor/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "FX Luminaire Luxor low voltage controller",
+  "config": {
```

Comment:
The options flow strings are missing here

Reply:
There isn't anything to configure, other than the host, which is in the strings.  Is there something else that you were referring to?  (There isn't even a username/password required.  The protocol is unauthenticated and is in plain text)